app.ts

```diff
@@ -244,17 +248,15 @@ export class App<
       })
     }
     mw.push({ type: 'mw', handler: this.notFound, path: '/' })
-
     const handle =
       (mw: Middleware<Req, Res>) =>
       async (req: Req, res: Res, next: NextFunction) => {
         const { handler, type, pattern } = mw
-
         const params =
           type === 'route' && pattern?.exec(req.url)?.pathname.groups || {}
 
         req.params = params as Record<string, string>
-
+        if (!req.path) req.path = req._urlObject.pathname
```

Comment:
req path is already set here

https://github.com/deno-libs/tinyhttp/blob/b48d0a06a924a7b01460fe91f21abd99acabf73f/extend.ts#L57